consensus/src/experimental/buffer_manager.rs

```diff
@@ -456,7 +454,7 @@ impl BufferManager {
                     let new_item = match item.add_signature_if_matched(*vote) {
                         Ok(()) => item.try_advance_to_aggregated(&self.verifier),
                         Err(e) => {
-                            error!("Failed to add commit vote {:?}", e);
+                            error!("Failed to add commit vote {:?} from author {:?} with commit info {:?}", e, author, commit_info);
```

Comment:
How about using structural logging? 

```
error!(
    vote = vote,
    author = author,
    commit_info = commit_info,
    "Failed to add commit vote."
);
```

Also, just curious, what's the difference between Display and Debug on these types?

Reply:
Good point - done.

>Also, just curious, what's the difference between Display and Debug on these types?

Not sure, I see the Display is printing the same information as Debug 😄.